### PR TITLE
add se resnet 152 profile script

### PR DESCRIPTION
**Pull request by jacquesqiao**

### CODE REVIEW

File: fluid/se_resnet_152/train.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                            fetch_list=[],
+                            use_program_cache=True)
+                else:
+                    exe.run(fluid.default_main_program(),
+                            feed=feed_dict,
+                            fetch_list=[],
+                            use_program_cache=True)
+                train_stop = time.time()
+                if step_id > args.skip_first_steps:
+                    train_time += train_stop - train_start
```
Comment (chengduoZH) [inline fluid/se_resnet_152/train.py]: Maybe the prof also needs warm up.

File: fluid/SE-ResNeXt-152/train.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+    #optimizer = fluid.optimizer.SGD(learning_rate=0.002)
+    optimizer = fluid.optimizer.Momentum(
+        learning_rate=fluid.layers.piecewise_decay(
+            boundaries=[100], values=[0.1, 0.2]),
+        momentum=0.9,
+        regularization=fluid.regularizer.L2Decay(1e-4))
+    opts = optimizer.minimize(avg_cost)
+
+    # fluid.memory_optimize(fluid.default_main_program())
```
Comment (chengduoZH) [inline fluid/SE-ResNeXt-152/train.py]: Why do you remove this? 
For big batch_size, such as 12, the model will be oom, if without memory_optimize.

Comment (jacquesqiao) [reply fluid/SE-ResNeXt-152/train.py]: add back, I think it's a option for user to test.

File: fluid/SE-ResNeXt-152/train.py [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    if not infer:
+        drop = fluid.layers.dropout(x=pool, dropout_prob=0.2)
+    else:
+        drop = pool
+    out = fluid.layers.fc(input=drop, size=class_dim, act='softmax')
+    return out
+
+
+def time_stamp():
+    return int(round(time.time() * 1000))
```
Comment (chengduoZH) [inline fluid/SE-ResNeXt-152/train.py]: What is `time_stamp` used to do?

File: fluid/SE-ResNeXt-152/train.py [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+    #optimizer = fluid.optimizer.SGD(learning_rate=0.002)
+    optimizer = fluid.optimizer.Momentum(
+        learning_rate=fluid.layers.piecewise_decay(
+            boundaries=[100], values=[0.1, 0.2]),
+        momentum=0.9,
+        regularization=fluid.regularizer.L2Decay(1e-4))
+    opts = optimizer.minimize(avg_cost)
+
+    fluid.memory_optimize(fluid.default_main_program())
```
Comment (chengduoZH) [inline fluid/SE-ResNeXt-152/train.py]: Maybe it is better that adding an argument, such as `use_memopt`.

**Pull request merged**